src/User/UserAuthantication.java

```diff
@@ -0,0 +1,34 @@
+package User;
+
+
+
+public class UserAuthantication {
+    private String Hashing (String pass){
```

Comment:
Any fields should be at the top of the class then comes the public methods then private methods ( if there is any ) 

1. Class instance variables 
2. Getter and setter methods 
3. Normal methods 
4. Privat methods 

@FCICB/batch3 Make sure all take care of these comments